doc/release-notes/release-7.sgml.in

```diff
@@ -57,7 +57,8 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing directives<label id="modifieddirectives">
 <p>
 <descrip>
-	<p>No changed directives in this version.
+	<tag>buffered_logs</tag>
+	<p>Support <em>off</em> to disable UDP log buffering.
```

Comment:
@yadij, please apply one of the suggestions (about the text you requested [earlier](https://github.com/squid-cache/squid/pull/1315#pullrequestreview-1361250254)) or resolve this change request without addressing it. Your earlier PR approval was auto-dismissed by your follow up commit [f9d0832](https://github.com/squid-cache/squid/pull/1315/commits/f9d083275a31e5c05fdae020b2f8a99347852baa), and I do not want to clear this PR for merging myself while there is a chance to improve this text...

While you are at it, please also consider using `"buffered_logs off"` instead of `buffered_logs OFF` in the PR title (i.e. the future commit message and possibly a ChangeLog entry) because Squid does not actually support `OFF` spelling in boolean configuration directive values.